web/src/tippyjs.js

```diff
@@ -105,6 +105,19 @@ export function initialize() {
         target: ".tippy-zulip-tooltip",
     });
 
+    delegate("body", {
+        target: ".tippy-username-tooltip",
+        placement: "top",
+        onShow(instance) {
+            instance.setContent(instance.reference.textContent);
+            instance.popper.querySelector(".tippy-content").classList.add("tooltip_username");
+        },
+        onHidden(instance) {
+            instance.destroy();
+        },
+        appendTo: () => document.body,
+    });
+
```

Comment:
Added a separate tooltip configuration to manage the tooltip displayed when hovering over a User's profile card. Previously, it used `data-tippy-content`, so applying CSS to it could potentially break other tooltips.